src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +26,16 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> arrayList = new ArrayList<>();
```

Comment:
Favour not using types in the parameter name. `products` is better because your collection is of Product objects